src/slave/flags.hpp

```diff
@@ -203,6 +203,8 @@ class Flags : public virtual logging::Flags
 #if ENABLE_XFS_DISK_ISOLATOR
   std::string xfs_project_range;
   bool xfs_kill_containers;
+  bool xfs_enable_quotas;
```

Comment:
I'm probably missing something but I can't see anything using `xfs_enable_quotas`?

Reply:
There isn't any usage, good catch. The port has been removed and a new patch (https://github.com/apache/mesos/pull/475) will serve to replace the intention behind the flag (i.e. enabling the enforcement of XFS quotas).